test/parallel/test-v8-features.js

```diff
@@ -0,0 +1,12 @@
+var common = require('../common');
+var assert = require('assert');
+var v8 = require('v8');
+
+eval('"use strict"; let x = 42');  // Should not throw.
+eval('"use strict"; const y = 42');  // Should not throw.
+assert.throws(function() { eval('"use strict"; class C {}'); });
+
+v8.setFlagsFromString('--harmony_classes');
+eval('"use strict"; class C {}');  // Should not throw.
+eval('"use strict"; let x = 42');  // Should not throw.
+eval('"use strict"; const y = 42');  // Should not throw.
```

Comment:
afaik `let` and `const` are included under another patch, is there a really good reason to have them here? (Also, I thought those worked without strict?)